editor/src/core/property-controls/property-controls-utils.ts

```diff
@@ -283,3 +306,68 @@ export function getPropertyControlsForTarget(
     return null
   }
 }
+
+export const PropertyControlsInfoIFrameID = 'property-controls-info-frame'
+
+let propertyControlsIFrameReady: boolean = false
+let lastPropertyControlsInfoSendID: number | undefined = undefined
+
+export function setPropertyControlsIFrameReady(value: boolean): void {
+  propertyControlsIFrameReady = value
+}
+
+export function sendPropertyControlsInfoRequest(
+  exportsInfo: ReadonlyArray<ExportsInfo>,
+  nodeModules: NodeModules,
+  projectContents: ProjectContents,
+): void {
+  sendToIFrame()
```

Comment:
I really dislike that this is called before the function is defined. Please move it down :)

Reply:
Done in 3f385e6b8.